src/test/compile-fail/if-let-arm-types.rs

```diff
@@ -13,6 +13,8 @@ fn main() {
         //~^ expected (), found integral variable
         //~| expected type `()`
         //~| found type `{integer}`
+        //~| NOTE: in this expansion of desugaring of `if let`
+        //~| NOTE: in this expansion of desugaring of `if let`
```

Comment:
These lines are required by compiletest, but not shown in rustc output (see the second commit).